### PR TITLE
add first documentation page for destination dynamic yield by mastercard

### PR DESCRIPTION
**Pull request by djluis**
### Proposed changes

Adds first documentation page for destination Dynamic yield by Mastercardard audiences

### Merge timing
This can be published after we complete the QA tests for Dynamic Yield by Mastercard destination.

### CODE REVIEW

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Dynamic Yield By Mastercard Audiences Destination
+beta: true
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:3]:
```suggestion
beta: true
id: 64ede9fe67158afa8de61480
engage: true
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+{% include content/plan-grid.md name=**actions** %}
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:6]:
```suggestion
{% include content/plan-grid.md name=actions %}
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+[Dynamic Yield by Mastercard](https://www.dynamicyield.com/){:target=**_blank”} helps businesses deliver digital customer experiences that are personalized, optimized, and synchronized.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:8]:
```suggestion
[Dynamic Yield by Mastercard](https://www.dynamicyield.com/){:target="_blank”} helps businesses deliver digital customer experiences that are personalized, optimized, and synchronized.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+With Dynamic Yield’s Experience OS, you can algorithmically match content, products, and offers to each individual customer to increase revenue, build customer loyalty, and gain a sustainable competitive advantage.
+
+This destination is maintained by Dynamic Yield by Mastercard. For any issues with the destination, [contact their Support team](mailto:info@dynamicyield.com).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:11]:
```suggestion
This destination is maintained by Dynamic Yield by Mastercard. For any issues with the destination, [contact the Dynamic Yield support team](mailto:info@dynamicyield.com).
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+## Getting started
+
+**Dynamic Yield by Mastercard Audiences** is an Audience Destination which must be first connected to an Engage Space before it can be connected to individual Engage Audiences. The steps below outline how to connect the Destination to an Engage Space and then to an Audience.
+
+### Creating an instance of the Dynamic Yield Destination, and connecting it to an Engage Space.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:17]:
```suggestion
### Create an instance of the Dynamic Yield Destination
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+1. In your Segment Workspace, navigate to Connections > Catalog
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:19]:
```suggestion
1. From your Segment workspace, navigate to **Connections > Catalog**
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+2. Use the search field to find the **Dynamic Yield by Mastercard Audiences** Destination, then select it.
+3. Click the **Add Destination** button
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:21]:
```suggestion
3. Click **Add destination**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+4. Select the Engage Space to connect to, then click the **Next button**. Note that you must select an Engage Space and not a regular Connections Space.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:22]:
```suggestion
4. Select the Engage Space you'd like to connect your destination to and click **Next**. 
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+5. Provide a name for the Destination then click **Create destination**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:23]:
```suggestion
5. Enter a name for your destination and click **Create destination**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+### Configuring Basic Settings
+1. Provide the Section ID and Connection Key on the Settings tab for your Dynamic Yield by Mastercard destination.
+2. Enable the Destination using the toggle, then click the **Save changes** button.
+
+
+### Create and configure a Mapping
+Before connecting your Audiences to the Destination you will need to create, configure and enable a Mapping.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:31]:
```suggestion
You must first create, configure, and enable a Mapping before connecting your Audiences to the Dynamic Yield by Mastercard destination.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Navigate to the Mappings tab in the Dynamic Yield by Mastercard Destination.
+2. Click the **New Mapping** button and select the **Sync Audience **Action**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:34]:
```suggestion
2. Click **New Mapping** and select the **Sync Audience Action**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+3. Ensure that the only condition to trigger the mapping is **Event Type is track** (remove the **Event Type is identify** condition if it is present).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:35]:
```suggestion
3. Ensure that the only condition to trigger the mapping is **Event Type is Track** (remove the **Event Type is Identify** condition, if present).
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+4. Click the Save button to save the Mapping configuration.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:36]:
```suggestion
4. Click **Save**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+5. You can then enable the Mapping from the Mappings tab using the **Status** toggle.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:37]:
```suggestion
5. Enable the Mapping from the Mappings tab using the **Status** toggle.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+Once these steps have been completed you can connect Audiences to the Destination.
+
+### Connecting Audiences to the Dynamic Yield by Mastercard Destination
+
+1. Navigate to your Engage Space > Audiences.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:43]:
```suggestion
1. Navigate to your **Engage Space > Audiences**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+2. Select the Audience you'd like to sync to Dynamic Yield
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:44]:
```suggestion
2. Select the Audience you'd like to sync to your Dynamic Yield by Mastercard Audiences destination. 
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+3. Click the **+ Add destination** button, then select the **Dynamic Yield by Mastercard Audiences** Destination you connected earlier. Click Add Destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:45]:
```suggestion
3. Click **+ Add destination**, select the **Dynamic Yield by Mastercard Audiences** Destination you connected earlier, and click **Add Destination.**
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+4. The Audience Settings panel will display for your Destination.
+5. Provide a value for the **Audience Name** field. This name will be used by Segment when creating the Audience in Dynamic Yield.
+6. Provide a value for the **Identifier Type** field. Valid entries are any one of **userid** or **anonymousid** or **email**. * See **Customized Identifier Setup** below for how to configure identifiers other than userid, email or anonymousid.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:48]:
```suggestion
4. On the Audience Settings panel, provide a value for the following fields: 
     - **Audience Name**: The name Segment uses when creating the Audience in Dynamic Yield.
     - **Identifier Type**: Select `userid`, `anonymousid`, or `email`. * See [Customized Identifier Setup](#customized-identifier-setup) for how to configure identifiers other than userid, email or anonymousid.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+6. Enable the **Send Track** toggle. You don't need to change the **Enter Event** or **Exit Event** fields as these are not used by this Destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:49]:
```suggestion
5. Enable the **Send Track** toggle. You don't need to change the **Enter Event** or **Exit Event** fields, as these are not used by this Destination.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+7. Click the **Default Setup** panel under **Event settings**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:50]:
```suggestion
6. Click **Default Setup** panel under **Event settings**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+8. Click the Save button to save the Audience Settings.
+9. Click the **Add Destination** button.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:52]:
```suggestion
7. Click **Save** and then click **Add Destination**.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+10. The Destination will now be connected to the Audience and will start syncing data to Dynamic Yield.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:53]:
```suggestion
The Destination is now connected to your Audience and starts syncing data to Dynamic Yield.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 23]
```diff
@@ -0,0 +1,67 @@
...
+
+
+### Customized Identifier Setup
+The Dynamic Yield Audience Destination can accept identifiers other than userId, anonymousId or email - however this requires some additional configuration steps when connecting the Audience to your **Dynamic Yield Audiences** Destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:57]:
```suggestion
The Dynamic Yield Audience Destination can accept identifiers other than userId, anonymousId or email. However, this requires some additional configuration steps when connecting the Audience to your Dynamic Yield Audiences Destination.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 24]
```diff
@@ -0,0 +1,67 @@
...
+
+1. When connecting your Audience to the **Dynamic Yield Audiences** Destination, select the **Customized Setup** panel under **Event Settings**.
+2. Click the **Add identifier** button, then select the identifier type you'd like to use.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:60]:
```suggestion
2. Click**Add identifier** then select the identifier type you'd like to use.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 25]
```diff
@@ -0,0 +1,67 @@
...
+3. Provide a name for the identifier in the **identifier in destination** field.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:61]:
```suggestion
3. Provide a name for the identifier in the **Identifier in destination** field.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+4. Scroll back up to the top of the Audience Settings panel and ensure that the **Identifier Type** field contains the identical name for the identifier you configured in the Customized Setup panel.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:62]:
```suggestion
4. Scroll back up to the top of the Audience Settings panel and ensure that the **Identifier Type** field contains the name of the identifier you configured in the Customized Setup panel.
```

File: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md [hunk 27]
```diff
@@ -0,0 +1,67 @@
...
+6. Click the Save button to save your Audience Settings.
+6. Click the **Add Destination** button.
+7. The Destination will now be connected to the Audience and will start syncing data to Dynamic Yield using the specified custom identifier.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md:65]:
```suggestion
6. Click **Save** and then click **Add Destination** button.

The Destination is now connected to your Audience and starts syncing data to Dynamic Yield with the specified custom identifier.
```

**Review by tcgilbert — APPROVED** [review]
lgtm once @forstisabella changes are made

**Comment by djluis** [comment]
Thanks for the feedback @forstisabella

**Review by tcgilbert — APPROVED** [review]

**Review by forstisabella — DISMISSED** [review]

**Review by tcgilbert — APPROVED** [review]

**Review by forstisabella — APPROVED** [review]

**Pull request merged**